npm-shrinkwrap.json

```diff
@@ -1,7395 +1,7258 @@
 {
   "name": "etcher",
   "version": "1.0.0",
+  "lockfileVersion": 1,
```

Comment:
Sure, I was wondering if we can rename `npm-shrinkwrap.json` to `package-lock.json`, so we always interact with that new filename convention instead.